mvrx/src/main/kotlin/com/airbnb/mvrx/PersistState.kt

```diff
@@ -128,7 +128,7 @@ fun <T : MavericksState> restorePersistedMavericksState(
     val fieldCount = constructor.parameterTypes.size
 
     // There is 1 bitmask for each block of 32 parameters.
-    val parameterBitMasks = IntArray(fieldCount / 32 + 1) { 0 }
+    val parameterBitMasks = IntArray(fieldCount / 32 + if (fieldCount % 32 != 0) 1 else 0) { 0 }
```

Comment:
might be simpler to round up to the next int `ceil(fieldCount/32.0).toInt()`

Reply:
Yeah, that was what I was looking for yesterday but could not come up with it. I'll update it.